kDriveCore/Data/Models/File.swift

```diff
@@ -521,6 +521,10 @@ public final class File: Object, Codable {
 
     public var isDownloaded: Bool {
         // Check that size on disk matches, if available
+        guard !isDirectory else {
```

Comment:
This prevents the `download` action from showing in Files.app.
Is is safe for main app ?